ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarGroupViewDemoController.swift

```diff
@@ -235,12 +248,12 @@ extension AvatarGroupViewDemoController: UITextFieldDelegate {
         let button = textField == maxAvatarsTextField ? maxAvatarButton : overflowCountButton
         if let count = UInt(text) {
             if textField == maxAvatarsTextField {
-                button.isEnabled = count > 0 && count != maxDisplayedAvatars
+                button.state.isDisabled = count <= 0 || count == maxDisplayedAvatars
```

Comment:
nit: Can we add wrappers in `UIHostingVontroller` it becomes hard to read the negative connotation while trying to deduce the meaning of the condition?

Reply:
You mean a wrapper function like "button.isDisabled = true" that calls into the state?

If so, we have explicitly decided to expose the state object as is to make it easier (less code written) when we add or update state properties. (Anand made this comment in the original button prototype PR).